xml/depl_require.xml

```diff
@@ -2116,10 +2116,14 @@
       <para>
        For more information, refer to the &haguide;, available at
        <ulink url="https://www.suse.com/documentation/sle_ha/"/>. Especially
-       read the following chapters: <citetitle>Network Device
-       Bonding</citetitle>, and <citetitle>Installation and Basic
-       Setup</citetitle> (section <citetitle>Defining the Communication
-       Channels</citetitle>).
+       read the following chapter: <citetitle>Network Device
+       Bonding</citetitle>.
+      </para>
+      <para>
+       Using a second communication channel (ring) in &corosync; (as an
+       alternative to network device bonding) is not supported yet in &productname;.
+       By default, &productname; uses <literal>eth0</literal> (admin network) for
```

Comment:
Do we explicitly want to mention `eth0`?  Don't some servers end up with interfaces like `em0` or other weird and wonderful names?  And anyway the admin network could be on any interface in theory.  I would have thought it would make more sense to only mention the admin network, or at least word it as:

    By default, &productname; uses the admin network (typically <literal>eth0</literal>)